packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/devices.js

```diff
@@ -6,50 +6,50 @@ import MockWebex from '@webex/test-helper-mock-webex';
 import dto from './wdm-dto';
 
 describe('plugin-devices', () => {
-  let webex;
-  let devices;
-
-  beforeEach('initialize webex with the devices plugin', () => {
-    webex = new MockWebex({
-      children: {
-        devices: Devices
-      }
-    });
+  describe('Devices', () => {
```

Comment:
These changes can be ignored as it was simply fixing an indention style due to an additional `describe()`.